handlers/handler-loadbalance/src/main/java/io/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -350,4 +350,16 @@ private void loadFilter(String filter, LoadBalancer lb) {
       }
     }
   }
+
+  private boolean isNotChanged(String p, String s) {
```

Comment:
maybe "isChanged" is better

Reply:
Okay, i will try

Reply:
oh, i think use "isChanged" will more complex